CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Manual curation
 - argannot_curation: (Tet)tetH:EF460464:6286-7839:1554 was incorrectly annotated as ARO:3004797 which is a beta-lactamase due to a loose RGI hit. This was manually curated to ARO:3000175.
+- deeparg, megares, resfinderfg & sarg curation: ARO:3004445 -> ARO:3005440, this was due to a change in the ARO and the ARO number for the RSA2 gene changing. **db_harmonisation must change to take this into account**
+> Incorrectly curated genes. Previously, they were directly mapped to drug classes. Correct parent ARO term has now been given.
```

Comment:
The `>` symbol has a specific meaning in markdown

Reply:
I have changed the formatting now. Is it ok?